improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -823,10 +823,10 @@ def test_error(self):
             "",
         ]
         msg = (
-            "Invalid condition chain found. First element has length > 1 ",
-            "but second element is not 'AND' or 'OR'.",
+            "Invalid condition chain found. First element has length > 1 "
+            "but second element is not 'AND' or 'OR'."
         )
-        with self.assertRaises(RuntimeError, msg=msg):
+        with self.assertRaisesRegex(RuntimeError, msg):
```

Comment:
Previously implemented incorrectly. Fixed in this PR, but unrelated to the change being implemented.